File: src/Backend/Domain/Auth.Entity/Ids4Entity/Clients/ClientPostLogoutRedirectUri.cs
using System;
using Dapper.Contrib.Plus;

namespace Auth.Entity.Ids4Entity
{
    /// <summary>
    /// 
    /// </summary>
    [Table("client_post_logout_redirect_uris")]
    public class ClientPostLogoutRedirectUri : SeedWork.Entity
    {
        /// <summary>
        /// Signout Url
        /// </summary>
        public string PostLogoutRedirectUri { get; set; }

        /// <summary>
        /// ClientId
        /// </summary>
        public Guid ClientId { get; set; }
    }
}

File: src/FrontEnd/auth-admin/src/types/entities/ids/client-post-logout-redirect-uri.ts
interface IClientPostLogoutRedirectUri {
  id?: string;
  postLogoutRedirectUri: string;
  clientId: string;
}

export default class ClientPostLogoutRedirectUri implements IClientPostLogoutRedirectUri {
  id?: string;
  postLogoutRedirectUri: string = '';
  clientId: string = '';
}